avocado/utils/network/interfaces.py

```diff
@@ -38,12 +38,10 @@
 class NetworkInterface:
     """
     This class represents a network card interface (NIC).
-
```

Comment:
Any reason to remove the empty line after the 'one-line' docstring?

If you look at the PEP-257, about multiple lines docstrings:

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a **blank line**, followed by a more elaborate description.

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings